AcademicReward/Database/GroupProfileRelationship.cs

```diff
@@ -50,5 +50,71 @@ public static ObservableCollection<Profile> getProfilesInGroup(Group group)
                 return null;
             }
         }
+
+        public static void addProfileToGroup(Profile profile, Group group)
+        {
+            try
+            {
+                int profileId = profile.ProfileID;
+                int groupId = group.GroupID;
+                using var con = new NpgsqlConnection(InitializeConnectionString());
+                con.Open();
+                //Select SQL query for getting all profiles
+                var sql = "INSERT INTO profilegroup (profileid, groupid) " +
+                    "VALUES (" + $"{profileId}" + ", " + $"{groupId}" + ");";
+                //Executing the query.
+                using var cmd = new NpgsqlCommand(sql, con);
+                cmd.ExecuteNonQuery();
+                //Closing the connection.
+                con.Close();
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine("Error while adding profile to group: {0}", e);
+            }
+        }
+
+        public static Profile findByUsername(string username)
+        {
+            try
+            {
+                using var con = new NpgsqlConnection(InitializeConnectionString());
```

Comment:
I guess I'm just curious, what happens if the username is not found? Or what if the username being added is an admin? What are we reporting back to the user?